src/test/java/ua/biz/test/pages/EmployeeListPage.java

```diff
@@ -12,15 +15,17 @@
     @FindBy(xpath = "//table[@id='resultTable']//tr[@class='odd' or @class='even']")
     private List<WebElement> emloyeeDataList;
 
-    public EmployeeListPage() {
-        super();
-    }
-
     public boolean isEmployeePresent(Employee employee) {
         return emloyeeDataList.stream().
                 map(WebElement::getText).
                 anyMatch(element -> element.contains(employee.getFirstName()) &&
                         element.contains(employee.getLastName()) &&
                         element.contains(String.valueOf(employee.getId())));
     }
+
+    public boolean isEmployeePresentByVacancy(Vacancy vacancy) {
+        Employee employee = EmployeeSQLDataProvider.getEmployeeByVacancy(vacancy);
```

Comment:
bad approach to use TestDataProviders in PageObjects cause these are two different layers

Reply:
I understood.